src/service_daemon.rs

```diff
@@ -2038,7 +2038,7 @@ impl Zeroconf {
         next_delay: u32,
         listener: Sender<ServiceEvent>,
     ) {
-        let addr_list: Vec<_> = self.intf_socks.keys().collect();
+        let addr_list: Vec<_> = self.intf_socks.keys().map(|itf| itf.addr.ip()).collect();
```

Comment:
Related to the general comment I gave, maybe we could print both IP and name by doing something like: 
```
let addr_list: Vec<_> = self.intf_socks.keys().map(|itf| (itf.ip(), itf.name)).collect();
```
(I didn't try locally, so just a thought)

Reply:
Yes it's simpler, but I personally find the <ip> (<itf-name>) more pleasant to read :)